lib/tasks/slack/admin.rake

```diff
@@ -0,0 +1,56 @@
+namespace :slack do
+  namespace :admin do
+    namespace :files do
+
+      desc "Prune files"
+      task prune: [:environment] do
+        puts "Pruning files..."
+
+        client = Slack::Web::Client.new
+
+        page = 1
+        pages = 1
+        count = 200
```

Comment:
Are there any special things about these numbers? If these could change, could they be passed in as parameters to the rake task? If they are constants, could they be frozen?

Reply:
It's unlikely they will change. We could freeze them, etc., but this will just be a periodic maintenance task so not sure it's worth it.